client/packages/electron/src/electron.ts

```diff
@@ -225,3 +202,55 @@ process.on('uncaughtException', error => {
         at process.processTimers (node:internal/timers:502:7)
   */
 });
+
+// App data store
+type StoreType = {
+  [key: string]: string | null;
+};
+const store = new ElectronStore<StoreType>();
+// /Users/andreievg/Library/Application Support/open mSupply
```

Comment:
comment not required?

Reply:
🙏 6a4f0f9c58f49cf2644206fd4996525db1b68231